test/config-generator.js

```diff
@@ -466,6 +465,33 @@ describe('The config-generator function', () => {
             const actualConfig = configGenerator(config);
             expect(actualConfig.devServer).to.be.undefined;
         });
+
+        it('devServer no hot mode', () => {
+            const config = createConfig();
+            config.runtimeConfig.useDevServer = true;
+            config.runtimeConfig.devServerUrl = 'http://localhost:8080/';
+            config.runtimeConfig.useHotModuleReplacement = false;
+            config.outputPath = '/tmp/public';
```

Comment:
pff, nevermind - I immediately saw the test failure after posting this. I'm trying a Windows path now to see if it'll keep things happy :)